projects/plugins/jetpack/extensions/blocks/blogroll/components/blogroll-appender/index.js

```diff
@@ -1,22 +1,20 @@
 import { Button, Popover } from '@wordpress/components';
 import { dispatch } from '@wordpress/data';
-import { useState, useEffect } from '@wordpress/element';
+import { useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import { plus } from '@wordpress/icons';
-import { debounce } from '../../../../shared/debounce';
```

Comment:
We should find a way to bring back debouncing especially now when we are hitting endpoints with search

Reply:
I already have logic in place to handle debouncing for the search input. 
I could not get this debounce function to work in the `use-get-site-details` useEffect, so I wrote my own logic for it.